group_vars/all/all.yml

```diff
@@ -2,11 +2,11 @@
 aws_region: "eu-west-1"
 # Current EKS Kubernetes version
 # https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
-k8s_version: "1.24"
+k8s_version: "1.30"
```

Comment:
Why not 1.31?

Reply:
1.30 is LTS I think?

Reply:
I don't think that's a distinction AWS make tbh, if we go 1.30 now, will have to do this again before July 28, 2025 to avoid fees. 1.31 would give an extra 6 months (although we should get in the habit of doing it frequently)